xmtp_mls/src/groups/mod.rs

```diff
@@ -272,6 +272,15 @@ where
         account_addresses_to_add: Vec<String>,
     ) -> Result<(), GroupError> {
         let account_addresses = sanitize_evm_addresses(account_addresses_to_add)?;
+        // get current number of users in group
```

Comment:
I'd love to add a test, is there a better way to add one other than creating 250 clients and adding them?

Reply:
You could investigate how to mock the `members()` method or creating a `members_count()` method to mock, but is the reason you don't want to use the real thing performance?

I'm suspecting the performance won't be that bad if you batch add 250 members, as opposed to adding them sequentially, because I believe that's still a single commit.

Reply:
I might do one for mocking and another for creating the clients
Just wasn't sure what performance hit would be like for a fairly simplistic test